src/kube-runtime/src/runtime.d/install_group.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+
+# Copyright (c) Microsoft Corporation
+# All rights reserved.
+#
+# MIT License
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated
+# documentation files (the "Software"), to deal in the Software without restriction, including without limitation
+# the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and
+# to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+# The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING
+# BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+# NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+# DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+PAI_WORK_DIR=/usr/local/pai
+CACHE_ROOT_DIR=${PAI_WORK_DIR}/package_cache
+
+function ubuntu_is_installed(){
+  for package in $1
+  do
+    dpkg -V $package &> /dev/null
+    if [ $? -ne 0 ]; then
+      return 1
+    fi
+  done
+  return 0
+}
+
+if [ $# -ne 1 ]; then
+  echo "Usage: bash -x install_group.sh <group_name>"
+  exit 1
+else
+  name=$1
+fi
```

Comment:
do we need to group packages by plugin name? some plugins may require the same package, then it will be cached multiple times in current implementation

```
try_to_install_by_cache(["package1", "package2"])
bash install_package.sh package1 package2
```
will this be better? and you can also remove `fallback_cmds` parameter

Reply:
This will not work since we must use the group name instead of multiple package names. If package names are used, the dependency order of these packages should also be calculated, which is not implemented.